src/Spice86/Views/DisassemblyView.axaml

```diff
@@ -1,252 +1,235 @@
+                        <MenuItem Header="Enable breakpoint"
+                                  Command="{Binding EnableBreakpointCommand}"
+                                  CommandParameter="{Binding SelectedDebuggerLine.Breakpoint, FallbackValue=null}"
+                                  IsVisible="{Binding SelectedDebuggerLine.Breakpoint, FallbackValue=null, Converter={x:Static ObjectConverters.IsNotNull}}"
+                                  IsEnabled="{Binding SelectedDebuggerLine.Breakpoint.IsEnabled, FallbackValue=false, Converter={x:Static BoolConverters.Not}}" />
+                        <Separator />
+                        <MenuItem Header="Move CS:IP here"
+                                  Command="{Binding MoveCsIpHereCommand}"
+                                  CommandParameter="{Binding SelectedDebuggerLine}" />
+                    </ContextMenu>
+                </ListBox.ContextMenu>
+                <ListBox.Styles>
+                    <Style Selector="ListBoxItem">
+                        <Setter Property="Padding" Value="4,1" />
+                        <Setter Property="Margin" Value="0" />
+                    </Style>
+                </ListBox.Styles>
+                <ListBox.ItemTemplate>
+                    <DataTemplate>
+                        <ContentControl FontFamily="Cascadia Code,Consolas,Menlo,Monaco,monospace"
+                                        behaviors:HighlightBehavior.IsContentHighlighted="{Binding IsCurrentInstruction}"
+                                        FontWeight="{Binding IsCurrentInstruction, Converter={x:Static converters:HighlightingConverter.TrueToBold}}"
+                                        Padding="0"
+                                        Margin="0"
+                                        behaviors:InstructionPointerBehavior.IsEnabled="True">
+                            <ContentControl.Styles>
+                                <Style Selector="SelectableTextBlock">
+                                    <Setter Property="Foreground"
+                                            Value="{Binding RelativeSource={RelativeSource AncestorType=ContentControl}, Path=Foreground}" />
+                                    <Setter Property="Background"
+                                            Value="{Binding RelativeSource={RelativeSource AncestorType=ContentControl}, Path=Background}" />
+                                    <Setter Property="behaviors:UseParentListBoxContextMenuBehavior.UseParentContextMenu" Value="True" />
+                                </Style>
+                            </ContentControl.Styles>
+
+                            <Grid ColumnDefinitions="Auto, Auto, *,4*,2*" Margin="0">
+                                <!-- Breakpoint indicator -->
+                                <Border Grid.Column="0"
+                                        Width="16"
```

Comment:
Use MinWidth and MinHeight instead of fixed Width and Height

Reply:
Done.